infoschema/tables_test.go

```diff
@@ -1138,6 +1138,95 @@ select * from t3;
 	user2.MustQuery("select user,query from `CLUSTER_SLOW_QUERY` order by query").Check(testkit.Rows("user2 select * from t2;", "user2 select * from t3;"))
 }
 
+func (s *testClusterTableSuite) TestSlowQuery(c *C) {
```

Comment:
Prefer to move the test to `executor` instead of keeping them in the `infoschema` package.

Reply:
done.